src/translationPipes/StripXMLTagsFromTranslation.ts

```diff
@@ -0,0 +1,15 @@
+import { Translation, TranslationPipe } from '../definitions.js'
+import { Language } from '../locales.js'
+
+export class StripXMLTagsFromTranslation implements TranslationPipe {
+    async execute(
+        _languagePack: Language,
+        _language: Language,
+        translation: Translation
+    ): Promise<Translation> {
+        return {
+            ...translation,
+            translation: translation.translation.replace(/<[^>]*>/g, ''),
```

Comment:
Im missing how the translation looks with XML tags - this would help to understand the code in the future

Reply:
Sure, I added an explaining comment :)